src/utilities/zAuctionRouting.ts

```diff
@@ -1,8 +1,11 @@
 import * as zAuction from "@zero-tech/zauction-sdk";
-import { zAuctionRoute } from "../types";
+import { zAuctionRoute, RouteUriToInstance } from "../types";
+import { Config } from "..";
 
 type GetDomainNameFromIdFunction = (domainId: string) => Promise<string>;
 
+const zAuctionRouteUriToInstance: RouteUriToInstance = {};
```

Comment:
This mapping needs to only exist *per* **ZNS** sdk instance.

The reason being is some undesirable behavior could happen if for ~some~ reason more than two ZNS SDK Instances are created at the same time.

We want to make each instance separate from the other, so please find a way of refactoring this so that multiple instance of ZNS SDK's don't share the same `zAuctionRouteUriToInstance`